nil pointer dereference for open command

Calling `riffraff open` fails with SIGSEGV

```
./riffraff open     
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x6ba520]

goroutine 1 [running]:
github.com/mre/riffraff/vendor/github.com/bndr/gojenkins.(*Jenkins).GetAllJobNames(0x0, 0x0, 0xc000146540, 0x1, 0x0, 0x1)
	/home/henry/go/src/github.com/mre/riffraff/vendor/github.com/bndr/gojenkins/jenkins.go:368 +0x40
github.com/mre/riffraff/job.FindMatchingJobs(0x0, 0x0, 0x0, 0x9, 0x0, 0xc000048180, 0x0, 0xc000178120)
	/home/henry/go/src/github.com/mre/riffraff/job/find.go:11 +0x40
github.com/mre/riffraff/commands.Open.Exec(0x0, 0x0, 0x0, 0x4, 0x0)
	/home/henry/go/src/github.com/mre/riffraff/commands/open.go:22 +0x5a
main.main()
	/home/henry/go/src/github.com/mre/riffraff/main.go:80 +0x37b
```

--- a/commands/open.go
+++ b/commands/open.go
@@ -15,7 +15,10 @@ type Open struct {
 }
 
 func NewOpen(jenkins *gojenkins.Jenkins, regex string) *Open {
-	return &Open{}
+	return &Open{
+		jenkins,
+		regex,
+	}
 }
 
 func (o Open) Exec() error {

